README.md

```diff
@@ -63,6 +65,17 @@ module "cloudfront" {
   origin_path                 = "/v1"
 }
 
+# Main cloudfront distribution with S3 origin with optional ACM certification and optional alias
+module "cloudfront" {
+  source                      = "voquis/cloudfront-s3-origin-acm/aws"
+  version                     = "0.0.3"
+  bucket_id                   = module.s3.s3_bucket.id
+  bucket_arn                  = module.s3.s3_bucket.arn
+  bucket_regional_domain_name = module.s3.s3_bucket.bucket_regional_domain_name
+  origin_path                 = "/examplefolder"
+  cloudfront_default_certificate = true
```

Comment:
Should we line these up to avoid terraform fmt complaining if someone copy/pastes.